README.md

```diff
@@ -29,96 +29,118 @@ npx chaperone-server --config chaperone.json
   - `registration_server` (optional) is an object describing what server to contact in order to translate the Registration Code entered during sign up into a Membrane Proof suitable for Holochain ([Read more](https://github.com/Holo-Host/holo-nixpkgs/tree/develop/overlays/holo-nixpkgs/holo-registration-service))
     - `url` (required)
     - `payload` (optional; defaults to `undefined`) is an arbitrary value that will be passed to the registration server as additional information
   - `skip_registration` (optional) if false or undefined, a registration code field is shown on the the sign up form. The behavior of this field depends on whether `registration_server `(above)` has been set. If `registration_server` is set, the registration code is sent to the registration server to exchange for a membrane proof. If `registration_server` is not set, the registration code entered is treated as a membrane proof itself and used directly in installing the happ.
 
 ```javascript
-const envoy = new Connection(
-  null,
-  (signal) => console.log("Got signal", signal),
-  {
+const websdk = new WebSDKAPI()
+const holo = await websdk.connect({
+  chaperone_url: null,
+  auth_form_customization: {
     logo_url: "my-logo.png",
     app_name: "My App",
     skip_registration: true,
+    anonymous_allowed: false
   }
-);
+});
 ```
 
-### `.ready( ) -> Promise<null>`
-
-Wait for the connection to be ready.
+```typescript
+type AgentInfo = {
+  id: string, // pub key
+  is_anonymous: boolean,
+  host_url: string
+}
 
-Asynchronous short-hand for connection event.
+type HostedAppInfo = {
+  is_connected: boolean,
+  hha_id: string, // note> added this here instead of making a separate call, make sure to match in chaperone
+  agent_info: AgentInfo
+}
 
-```javascript
-.on('connected', () => { fulfill(); });
+type InstalledCell = {
+  cell_id: CellId
+  role_id: RoleId
+};
+
+type DisabledAppReason =
+  | { never_started: null }
+  | { user: null }
+  | { error: string }
+
+type InstalledAppInfoStatus = 
+  | { disabled: { reason: DisabledAppReason } }
+  | { paused: { reason: { error: string } } }
+  | { running: null }
+
+type InstalledAppInfo = {
+  installed_app_id: string
+  cell_data: Array<InstalledCell>
+  status: InstalledAppInfoStatus
+}
 ```
 
-### `.agentInfo() -> Promise<object>`
-
-Returns an object with properties of an agent created in chaperone:
-
-```
-{
-  anonymous: boolean,
-  agent_id: string, // in multicodec encoding
-  agent_id_base64: string // base64 encoded
-}
+### `.ready() -> Promise<null>`
+```javascript
+.on('available', () => { fulfill() });
 ```
+Wait for the app to be ready.
+Asynchronous short-hand for available event.
 
-### `.zomeCall( dna_handle, zome_name, function_name, args ) -> Promise<any>`
+### `.callZome( dna_handle, zome_name, function_name, args ) -> Promise<any>`
```

Comment:
@JettTech  - sorry, I forgot to update you: I spoke with @jost-s on Friday and he's happy to rename `callZome` to `zomeCall` in `client-js`, so I think we can keep this as `zomeCall`, and then very soon it will be called the same thing across the whole project 😅